api/server/middleware/audit_linux.go

```diff
@@ -279,12 +283,14 @@ func logAuditlog(c *container.Container, action string, username string, loginui
 	hostname := "?"
 	user := "?"
 	auid := "?"
+	ctr_id_short := "?"
 
 	if c != nil {
 		vm = c.Config.Image
 		vmPid = fmt.Sprint(c.State.Pid)
 		exe = c.Path
 		hostname = c.Config.Hostname
+		ctr_id_short = c.ID[0:12]
```

Comment:
Is c.ID guaranteed to be at least 12 characters by this time and place?  Otherwise, this would cause a panic.

Reply:
I believe it's guaranteed to be 12+ characters, but I'm not 100% sure.  I've added a length check.  Nice catch, ty!